karma.conf.js

```diff
@@ -0,0 +1,83 @@
+// Karma configuration
+// Generated on Mon Jun 22 2015 16:55:41 GMT-0400 (EDT)
+
+module.exports = function (config) {
+  "use strict";
+
+  config.set({
+
```

Comment:
Small nit: I realize this is probably generated, so feel free to ignore, but maybe just use one newline here, since every other object literal in this file is written without the extra whitespace preceding it's contents.